snippets/facets.liquid

```diff
@@ -459,7 +459,7 @@
         <facet-remove>
           <a href="{{ value.url_to_remove }}" class="active-facets__button active-facets__button--light">
             <span class="active-facets__button-inner button button--tertiary">
-              {{ value.label | escape }}
+              {{ filter.label }}: {{ value.label | escape }}
               {% render 'icon-close-small' %}
               <span class="visually-hidden">{{ 'products.facets.clear_filter' | t }}</span>
             </span>
```

Comment:
If we add the option for a label, should it also be applied to the "price" filter?

Reply:
We figure seeing `Price:` is not needed:

![image](https://user-images.githubusercontent.com/143817/157489603-3a60fea2-ca0e-48c5-98d5-b511fdf90401.png)

The same could be said for the Availability filter.

The Price, Availability, Product Type, and Brand filters are default filters you can only have one of each. Their value is self-explanatory.

For custom filters, it might not be as obvious.

Imagine product options or metafield filters for dimensions, for products that have multiple dimensions e.g. Inseam, waist size. The value alone, without the label, is confusing:
 
![image](https://user-images.githubusercontent.com/143817/157490404-5ce020f1-805c-4452-8a33-18eb68afdee3.png)

A blanket theme setting — always show, never show — is not ideal. Maybe we can be smart about this.

- Show label for all production options and metafield-based settings.
- No label for:
    - Product Type
    - Brand
    - Availability
    - Price